index.js

```diff
@@ -3,11 +3,17 @@
 const semver = require('semver')
 const console = require('console')
 
+const DISPLAY_NAME_SYMBOL = Symbol.for('fastify-plugin.display-name')
```

Comment:
actually this is better as `Symbol.for('fastify.display-name')`, and _we look for it_ in https://github.com/fastify/fastify/pull/861, and we document it as part as https://github.com/fastify/fastify/blob/master/docs/Plugins.md.

I would not export the symbol from this module then.